endpoints/openrtb2/amp_auction.go

```diff
@@ -446,6 +446,10 @@ func (deps *endpointDeps) overrideWithParams(ampParams amp.Params, req *openrtb2
 		req.Imp[0].TagID = ampParams.Slot
 	}
 
+	if err := setConsentedProviders(req, ampParams); err != nil {
+		return []error{err}
```

Comment:
This error return isn't covered by tests, is it possible to cover/trigger this?

Reply:
Added `TestOverrideWithParams` to add coverage to various uncovered scenarios in the `overrideWithParams` function.